### PR TITLE
Second level imports

### PR DESCRIPTION
**Pull request by mbkumar**
This PR imports most of the classes and functions defined in submodules at third level to the module level at second level and minimizes the verbosity of the import statements.

For example `import simsopt.geo.curve.Curve` can now be imported as `import simsopt.geo.Curve`. Some of the classes and functions especially those that are infrequently used may still need to be imported from their submodules.

Please let me know if any important class or function is missed.

### CODE REVIEW

**Review by landreman — COMMENTED** [review]
Great, thanks for working on this.

It looks like the API docs list functions with the long names that include submodules. Is it possible to change the docs so classes and functions are listed with the new shorter names?

**Review by florianwechsung — DISMISSED** [review]

File: examples/2_Intermediate/stage_two_optimization.py [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
 from scipy.optimize import minimize
-from simsopt._core.optimizable import Weight
-from simsopt.geo.surfacerzfourier import SurfaceRZFourier
-from simsopt.objectives.fluxobjective import SquaredFlux
-from simsopt.objectives.utilities import QuadraticPenalty
-from simsopt.geo.curve import curves_to_vtk, create_equally_spaced_curves
-from simsopt.field.biotsavart import BiotSavart
-from simsopt.field.coil import Current, coils_via_symmetries
-from simsopt.geo.curveobjectives import CurveLength, CurveCurveDistance, \
+from simsopt._core import Weight
```
[the remark this answers is not in the document]
Comment (mbkumar) [reply examples/2_Intermediate/stage_two_optimization.py]: Done

**Comment by landreman** [comment]
When I build the docs for this branch, there are a ton of warnings like this:
~~~~
/Users/matt/simsopt/simsopt/src/simsopt/mhd/__init__.py:docstring of simsopt.mhd:1: WARNING: duplicate object description of simsopt.mhd, other instance in simsopt.mhd, use :noindex: for one of them
/Users/matt/simsopt/simsopt/src/simsopt/mhd/boozer.py:docstring of simsopt.mhd.boozer.Boozer:1: WARNING: duplicate object description of simsopt.mhd.Boozer, other instance in simsopt.mhd, use :noindex: for one of them
...
~~~~
Is there a way to suppress these?

**Comment by mbkumar** [comment]
If you delete the build folder inside the docs, these warnings go away.

Bharat Medasani

Engineer
Princeton Plasma Physics Lab (PPPL)

**Comment by landreman** [comment]
> If you delete the build folder inside the docs, these warnings go away. Bharat Medasani Engineer Princeton Plasma Physics Lab (PPPL)

Even after deleting the `docs/build` folder, when I then run `make html` I get all those warnings.

**Comment by mbkumar** [comment]
simsopt.mhd.rst had duplicate module level API doc generation. I removed
it. Now those warnings should go away.
Thanks for catching this.

Bharat Medasani

Engineer
Princeton Plasma Physics Lab (PPPL)

**Review by landreman — APPROVED** [review]

**Pull request merged**